netlify/netlify-plugin-nextjs
# fix: add next.js 12's new /trace endpoint to HIDDEN_PATHS

### Summary
The location of the `trace` file generated by next.js's build changed from `/traces/trace` to `/trace` in [next.js v12](https://github.com/vercel/next.js/releases/tag/v12.0.0) (from [this PR in particular](https://github.com/vercel/next.js/pull/28728)). 

Currently, going to `/trace` on a nextjs v12 + netlify site serves the `trace` file like any other static file.  This PR prevents that in the same way it also prevents access to `/traces`.

### Test plan

1. Visit the Deploy Preview's `/trace` page https://deploy-preview-1259--netlify-plugin-nextjs-demo.netlify.app/trace
2. Observe that it shows a 404 page similar to the pre-existing `/traces` page https://deploy-preview-1259--netlify-plugin-nextjs-demo.netlify.app/traces

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
https://github.com/vercel/next.js/pull/28728

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
